json-schema/json-schema-generic/src/main/scala/endpoints/generic/annotations.scala

```diff
@@ -1,12 +1,12 @@
 package endpoints.generic
 
 /**
-  * Documents a case class field.
+  * Adds a description to a case class field, a case class, or a sealed trait.
   *
-  * Annotate a case class field with this annotation to define its
-  * documentation.
+  * Annotate a case class field, case class, or sealed trait with this
+  * annotation to define its the description of the schema or record field.
```

Comment:
```suggestion
  * annotation to define its description.
```

Reply:
```suggestion
  * annotation to define the description of the schema or the description of the record field.
```
? Sounds heavy, no?

Reply:
Oof yeah - my docstring does not read well. How about:

```
Annotate a case class field, case class, or sealed trait with this
annotation to set a description for the schema or the record field.
```

Reply:
Yes, that one is great!

Reply:
I've updated this doc (as well as tweaking the mention of `@docs` in `documentation/`). Let me know if there is anything else you think we should change.